python/ray/tune/suggest/repeater.py

```diff
@@ -0,0 +1,151 @@
+import copy
+import itertools
+import logging
+import numpy as np
+
+from ray.tune.suggest.suggestion import SuggestionAlgorithm
+from ray.tune.experiment import convert_to_experiment_list
+
+logger = logging.getLogger(__name__)
+
+TRIAL_INDEX = "__trial_index__"
```

Comment:
hmm consider moving to `result.py`? no strong preference either way

Reply:
hm, for now i'll leave it here because it's not used in other places.